src/main/java/frc/robot/maps/GastonMap.java

```diff
@@ -0,0 +1,73 @@
+package frc.robot.maps;
+
+import com.chopshop166.chopshoplib.drive.SDSSwerveModule;
+import com.chopshop166.chopshoplib.motors.PIDSparkMax;
+import com.chopshop166.chopshoplib.sensors.PigeonGyro;
+import com.ctre.phoenix.sensors.AbsoluteSensorRange;
+import com.ctre.phoenix.sensors.CANCoder;
+import com.ctre.phoenix.sensors.PigeonIMU;
+import com.revrobotics.CANSparkMaxLowLevel.MotorType;
+
+import edu.wpi.first.math.geometry.Translation2d;
+import edu.wpi.first.math.util.Units;
+import edu.wpi.first.wpilibj.interfaces.Gyro;
+
+public class GastonMap extends RobotMap {
+    @Override
+    public DriveMap getDriveMap() {
+
+        // Value taken from CAD as offset from center of module base pulley to center of
+        // robot
+        final double MODULE_OFFSET_XY = 0; // TODO Get this value
+
+        // All Distances are in Meters
+        // Front Left Module
+        final CANCoder encoderFL = new CANCoder(1); // TODO Get cancoder id
+        encoderFL.configMagnetOffset(0); // TODO Get Magnet Offset
+        encoderFL.configAbsoluteSensorRange(AbsoluteSensorRange.Unsigned_0_to_360);
+        final SDSSwerveModule frontLeft = new SDSSwerveModule(new Translation2d(MODULE_OFFSET_XY, MODULE_OFFSET_XY),
+                encoderFL,
+                new PIDSparkMax(2, MotorType.kBrushless), new PIDSparkMax(1, MotorType.kBrushless),
+                SDSSwerveModule.MK4_V2); // TODO Change controller ids
+
+        // final Translation2d moduleLocation, final CANCoder steeringEncoder,
+        // final PIDSparkMax steeringController, final PIDSparkMax driveController,
+        // final Configuration conf
+
+        // Front Right Module
+        final CANCoder encoderFR = new CANCoder(2); // TODO Get cancoder id
+        encoderFR.configMagnetOffset(0); // TODO Get Magnet Offset
+        encoderFR.configAbsoluteSensorRange(AbsoluteSensorRange.Unsigned_0_to_360);
+        final SDSSwerveModule frontRight = new SDSSwerveModule(new Translation2d(MODULE_OFFSET_XY, MODULE_OFFSET_XY),
```

Comment:
The robot is going to be symmetrical so we can use the same offset. But some of these values need to be negative to represent the correct locations.